pyrobosim_msgs/srv/ResetWorld.srv

```diff
@@ -0,0 +1,10 @@
+# ROS service to reset the world state.
+
+---
+
+# Response
+# bool success
+# Indicates whether the world reset operation was successful:
```

Comment:
You can probably just keep this line and not the other 2.  It's pretty clear what this means.